test/v1/route.go

```diff
@@ -116,9 +117,21 @@ func IsRouteNotReady(r *v1.Route) (bool, error) {
 
 // RetryingRouteInconsistency retries common requests seen when creating a new route
 func RetryingRouteInconsistency(innerCheck spoof.ResponseChecker) spoof.ResponseChecker {
+	const neededSuccesses = 5
+	var successes int
 	return func(resp *spoof.Response) (bool, error) {
+		if resp.StatusCode == http.StatusNotFound || resp.StatusCode == http.StatusServiceUnavailable {
```

Comment:
These conditions have been removed here a while ago, do we really want to bring them back?

**Note to others:** We need this downstream because our ingress is fronted by Openshift Routes, which give our services proper URLs (so we can run with --resolvabledomain). That adds some inconsistency though which we need to retry. We currently carry a patch that readds these conditions just downstream.

Reply:
Probably not. I'll put it back